### ISSUE TITLE
FNX-519 ⁃ [Bug] History URL is cut with a large font set

### ISSUE THREAD

**Issue by bsurd** [mozilla-mobile/fenix#2294]
## Steps to reproduce
1. Set the device font size to the largest setting.
1. Open Fenix > 3dot menu > Your Library > History.
1. Observe the history entries.

### Expected behavior
There are no layout issues.

### Actual behavior
The URL for the history entries is cut, only the top part is visible.

### Device information
* Depending on the available font size options from one device to another, some don't display the URL at all. (eg. Samsung device)
* Android device: 
  * Samsung Galaxy S8 (Android 9)
  * Google Pixel (Android 9)
* Fenix version: 5/6
![1](https://user-images.githubusercontent.com/17178945/57232732-e6375e80-7025-11e9-91f7-05bbe4c32c78.png)
![screen](https://user-images.githubusercontent.com/17178945/57232736-e9cae580-7025-11e9-9dce-789985a608b3.png)

**Comment by vesta0** [comment]
@eeejay we currently have 3 bugs related to larger font size (will be tagging you in all of them). Let us know what your recommended approach would be.

**Comment by ekager** [comment]
Seems like if we made the `minHeight` what the current `height` for these rows are and then made height `wrapContent` (or something along those lines) we could be okay in this list (and other lists) with not much effort.

**Comment by ekager** [comment]
Looks like these changes should impact bookmarks and history since they share a layout file `library_site_item`

**Comment by ekager** [comment]
@vespertineSiren is going to pick up this issue

**Comment by vespertineSiren** [comment]
I would like to be assigned this :)

**Comment by ekager** [comment]
@vespertineSiren and I cannot reproduce this anymore, so adding a QA needed

**Comment by AndiAJ** [comment]
Hi @ekager I've just re-checked on the latest Nightly Build #13190606 from 11/15 using the following devices:
• Google Pixel 3a XL (Android 9)
• Samsung Galaxy Note 9 (Android 9)
• Huawei Mate 20 Lite (Android 8.1.0) 
• Samsung Galaxy Note 9 (Android 8.1.0)
 
<details>
    <summary>✔️ Works on the following devices (Dropdown with examples) </summary>

► **Google Pixel 3a XL (Android 9)**
![rsz_screenshot_20191115-120403](https://user-images.githubusercontent.com/51314259/68935817-e026f500-07a1-11ea-8530-be948f35e0ee.png)

► **Huawei Mate 20 Lite (Android 8.1.0)**
![rsz_screenshot_20191115-113907](https://user-images.githubusercontent.com/51314259/68935724-b4a40a80-07a1-11ea-96d3-939d51bf4cc4.jpg)


</details>


<details>
    <summary>❌ Still reproducible on the following devices (Dropdown with examples)</summary>

► **Samsung Galaxy Note 9 (Android 9)**
![rsz_screenshot_20191115-113453_firefox_nightly (1)](https://user-images.githubusercontent.com/51314259/68936126-8bd04500-07a2-11ea-96aa-5973b98bf89a.jpg)


► **Samsung Galaxy Note 9 (Android 8.1.0)**
![rsz_screenshot_20191115-113646_firefox_nightly](https://user-images.githubusercontent.com/51314259/68935500-65f67080-07a1-11ea-8ebe-82e4cfe8d823.jpg)

</details>

**I'll remove the QA needed label until further clarifications.**

**Comment by vespertineSiren** [comment]
Can I be assigned this? I've implemented @ekager fix above. The tests are running smoothly for non-Samsung devices. I'm currently waiting for Samsung devices to be available on samsung RTL to test on those devices mentioned in @AndiAJ 's post.

**Comment by AndiAJ** [comment]
Hi, I've just checked on the latest **Nightly Build #13240608 from 20/11** and this is **still reproducible** on the following devices:
 ❌ **Samsung Galaxy Note 9 (Android 9)**
![rsz_screenshot_20191120-113056_firefox_nightly](https://user-images.githubusercontent.com/51314259/69227213-2a391d80-0b8a-11ea-83e8-c2a15d37d0db.jpg)

 ❌ **Samsung Galaxy Note 9 (Android 8.1.0)**
![rsz_screenshot_20191120-113408_firefox_nightly](https://user-images.githubusercontent.com/51314259/69227304-4a68dc80-0b8a-11ea-971a-0eee64bd7bd5.jpg)

**Comment by ekager** [comment]
Let's try this again! :)

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the latest **Nightly Build #13261307 from 11/22** using the following devices:
• Google Pixel 3a XL (Android 9)
• Samsung Galaxy S10+ (Android 9)
• Samsung Galaxy Note 9 (Android 9)
• Huawei Mate 20 Lite (Android 8.1.0)
• Samsung Galaxy Note 9 (Android 8.1.0)

<details>
    <summary>Screenshots from the problematic devices (Dropdown)</summary>
 
► Samsung Galaxy S10+ (Android 9)
![rsz_screenshot_20191122-171803_firefox_nightly](https://user-images.githubusercontent.com/51314259/69438609-032b4900-0d4e-11ea-9103-709485c469c5.jpg)

► Samsung Galaxy Note 9 (Android 9)
![rsz_screenshot_20191122-171854_firefox_nightly](https://user-images.githubusercontent.com/51314259/69438727-3a99f580-0d4e-11ea-90f5-38b5d8eeafa8.jpg)

 ► Samsung Galaxy Note 9 (Android 8.1.0)
![rsz_screenshot_20191122-171814_firefox_nightly](https://user-images.githubusercontent.com/51314259/69438772-569d9700-0d4e-11ea-94e9-d46c9fd1d2d4.jpg)


</details>
